build.sbt

```diff
@@ -647,7 +647,7 @@ lazy val scalaVersionAgnosticCommonSettings = Seq(
 ) ++ releaseSettings
 
 lazy val commonSettings = scalaVersionAgnosticCommonSettings ++ Seq(
-  scalaVersion := "2.12.1",
+  scalaVersion := "2.10.6",
```

Comment:
I'd prefer to stick either to `2.11.8` (for the `cli` or scala-js only modules to be fine in particular) or `2.12.1` (to which the former modules will switch to eventually) rather than `2.10.x`.